language-server/dm/executionManager.ml

```diff
@@ -569,6 +581,33 @@ let get_context st id =
     let sigma = Evd.from_env env in
     Some (sigma, env)
 
+(*
+    match find_fulfilled_opt id st.of_sentence with
+    | Some (Success (Some { interp = ist; synterp = sst })) when Option.has_some ist.lemmas ->
+      Vernacstate.Interp.unfreeze_interp_state ist;
+      Vernacstate.Synterp.unfreeze sst;
+      let proof = get_proofview st pos in
+      None
+    | _ -> log "Cannot unfreeze state";
+    None   
+*)
+
+let get_completions st id =
+  let aux () = 
+    match find_fulfilled_opt id st.of_sentence with
+    | Some (Success (Some { interp = interp_state; synterp = synterp_state })) when Option.has_some interp_state.lemmas ->
+      Vernacstate.Interp.unfreeze_interp_state interp_state;
```

Comment:
Why do you need to imperatively unfreeze the system state here?

Reply:
Because of this issue https://github.com/coq-community/vscoq/issues/458